src/Pkg.jl

```diff
@@ -296,7 +296,8 @@ const build = API.build
     Pkg.pin(pkgs::Union{PackageSpec, Vector{PackageSpec}}; io::IO=stderr, all_pkgs::Bool=false)
 
 Pin a package to the current version (or the one given in the `PackageSpec`) or to a certain
-git revision. A pinned package is never updated. To pin all dependencies set `all_pkgs=true`.
+git revision. A pinned package is never updated: if `pkg` is tracking a path, or a repository,
+those remain tracked.
```

Comment:
Did you mean to remove the info about pinning all?

Also the new sentence seems confusing. "Never updated" and "remain tracked" seem like they mean opposite things?

Reply:
What is meant is that an update is not automatically updated. (If you look at the previous sentence "A pinned package is never updated." it was also confusing before).